src/Microsoft.Health.Dicom.Core/Exceptions/DicomImageException.cs

```diff
@@ -0,0 +1,14 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Exceptions;
+
+public class DicomImageException : DicomServerException
+{
+    public DicomImageException()
```

Comment:
Why are we creating this exception? I thought this is thrown from fo-dicom library

Reply:
Fo dicom creates a separate exception i catch it and throw this one